interceptor/tracing_interceptor.go

```diff
@@ -41,6 +42,9 @@ const (
 // Tracer is an interface for tracing implementations as used by
 // NewTracingInterceptor. Most callers do not use this directly, but rather use
 // the opentracing or opentelemetry packages.
+//
+// All implementations must embed BaseTracer to safely
+// handle future changes.
 type Tracer interface {
```

Comment:
By the way, we really appreciate this being a public interface. I think this is going to significantly simplify our Datadog tracing implementation (and make us more confident in open sourcing a supported Temporal/Datadog integration if we can ensure that the module in which this interface is defined follows semver 😉)

Reply:
:+1: Will keep it exported then.